blocks_reexecutor/blocks_reexecutor.go

```diff
@@ -42,10 +42,9 @@ func (c *Config) Validate() error {
 }
 
 var DefaultConfig = Config{
-	Enable:          false,
-	Mode:            "random",
-	Room:            runtime.NumCPU(),
-	BlocksPerThread: 10000,
+	Enable: false,
+	Mode:   "random",
+	Room:   runtime.NumCPU(),
```

Comment:
if we don't already check it, we should check in `Config.Validate` if `Room > 0` (I think we only do `Room != 0` there).